config/sync/user.role.site_builder.yml

```diff
@@ -34,7 +34,6 @@ permissions:
   - 'administer crop types'
   - 'administer display modes'
   - 'administer entity browsers'
-  - 'administer footer menu items'
```

Comment:
We are not using this footer menu. We have a global footer and a local footer that uses other configurations.